### DISCUSSION TITLE
Optimizing over Multiple Surrogate Models

### DISCUSSION THREAD

**Discussion by jeanstgo** [snl-dakota/dakota#85] [Q&A]
Greetings,

I am looking to run a multiple-objective global optimization using surrogates to calibrate an FE model to experimental data. The strategy is to use a gaussina_process to build my surrogate models and MOGA for my optimizer.  The main catch is that I need to build a surrogate for each objective since I am calibrating the FE model to three different types of experiments. How would I write a Dakota input script such that I can perform optimization using more than one surrogate? 

Thanks

**Comment by jadamstephens** [comment]
Hi,

There's no trick to it. By default, Dakota will build a separate gaussian process for each calibration term. You just need one surrogate model specification, and Dakota will use it for all responses.

Adam
